DeadCrumbs/app/src/main/java/dead/crumbs/ui/MainActivity.kt

```diff
@@ -70,6 +75,36 @@ class MainActivity : AppCompatActivity() {
         }
     }
 
+    //------------Dead Reckoning-------------//
+
+    private lateinit var drService: DeadReckoningService
+    private var drBound: Boolean = false
+
+    private fun startDeadReckoning(){
+        val intent = Intent(this, DeadReckoningService::class.java)
+        startService(intent)
+        Intent(this, DeadReckoningService::class.java).also { intent ->
+            bindService(intent, connectionDeadReckoningService, Context.BIND_AUTO_CREATE)
+        }
+    }
+
+
+
+    /** Defines callbacks for service binding, passed to bindService()  */
+    private val connectionDeadReckoningService = object : ServiceConnection {
+
+        override fun onServiceConnected(className: ComponentName, service: IBinder) {
+            // We've bound to DeadReckoningService, cast the IBinder and get LocalService instance
+            val drBinder = service as DeadReckoningService.LocalBinder
+            drService = drBinder.getService()
+            drBound = true
+        }
+
+        override fun onServiceDisconnected(arg0: ComponentName) {
+            drBound = false
```

Comment:
This is something me and Bjarke included in the Bluetooth counterpart but is unnecessary/obsolete. Replace with //Do nothing.

Reply:
Done